contracts/core/Registry.sol

```diff
@@ -49,10 +49,6 @@ contract Registry is IRegistry, Initializable, Native, AccessControlUpgradeable,
     /// @dev Profile.id -> Profile
     mapping(bytes32 => Profile) public profilesById;
 
-    /// @notice This maps the profile ID to the pending owner
-    /// @dev Profile.id -> pending owner
-    mapping(bytes32 => address) public profileIdToPendingOwner;
```

Comment:
Being this an upgradeable contract, should we remove the `profileIdToPendingOwner` declaration? It would be easier to review future upgrades if we leave it as private and change its name to something like `profileIdToPendingOwner_deprecated`.